packages/itwinui-react/src/core/ComboBox/ComboBoxMenuItem.tsx

```diff
@@ -2,12 +2,15 @@
  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
-import cx from 'classnames';
 import * as React from 'react';
 import type { MenuItemProps } from '../Menu/MenuItem.js';
-import { useSafeContext, useMergedRefs, Box } from '../utils/index.js';
-import type { PolymorphicForwardRefComponent } from '../utils/index.js';
+import {
+  useSafeContext,
+  useMergedRefs,
+  type PolymorphicForwardRefComponent,
```

Comment:
can you move this into a separate `import type` statement?

see https://github.com/iTwin/iTwinUI-react/pull/996